login-workflow/src/new-architecure/components/workflow/WorkflowCardActions.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import CardActions from '@mui/material/CardActions';
+import MobileStepper from '@mui/material/MobileStepper';
+import Button from '@mui/material/Button';
+import { WorkflowCardActionsProps } from './WorkflowCard.types';
+import { cx } from '@emotion/css';
+import { getWorkflowCardActionsUtilityClass, WorkflowCardActionsClassKey } from './Utility';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+const useUtilityClasses = (ownerState: WorkflowCardActionsProps): Record<WorkflowCardActionsClassKey, string> => {
+    const { classes } = ownerState;
+
+    const slots = {
+        root: ['root'],
+        nextButton: ['nextButton'],
+        previousButton: ['previousButton'],
+        stepper: ['stepper'],
+    };
+
+    return composeClasses(slots, getWorkflowCardActionsUtilityClass, classes);
+};
+
+export const WorkflowCardActions: React.FC<WorkflowCardActionsProps> = (props) => {
+    const {
+        canGoNext,
+        canGoPrevious,
+        showPrevious,
+        showNext,
+        previousLabel,
+        nextLabel,
+        onPrevious,
+        onNext,
+        currentStep,
+        totalSteps,
+        fullWidthButton,
+        sx,
+        ...cardActionsProps
+    } = props;
+    const defaultClasses = useUtilityClasses(props);
+
+    return (
+        <CardActions
+            sx={[{ justifyContent: 'flex-end', p: { sm: 2, md: 3 } }, ...(Array.isArray(sx) ? sx : [sx])]}
+            className={cx(defaultClasses.root)}
```

Comment:
cx is not needed when there is a single class...this'll be the last time I mention that in comments, but it applies to everywhere this occurs.